setup.py

```diff
@@ -3,8 +3,8 @@
 except ImportError:
     from distutils.core import setup
 
-__author__ = "The PredictionIO Team"
-__email__ = "help@tappingstone.com"
+__author__ = "Apache PredictionIO"
+__email__ = "user@predictionio.incubator.apache.org"
 __copyright__ = "Copyright 2014, TappingStone, Inc."
```

Comment:
@dszeto Can we replace copyright with ASF?
If yes, I'll update it and LICENSE.txt.

Reply:
Yes we should. In fact, we should also change all other SDKs to reflect that.

Thanks for taking care of this!